packages/core-components/src/components/OAuthRequestDialog/OAuthRequestDialog.tsx

```diff
@@ -89,7 +89,11 @@ export function OAuthRequestDialog(_props: {}) {
           classes={{ root: classes.title }}
           id="oauth-req-dialog-title"
         >
-          <Typography className={classes.titleHeading} variant="h1">
+          <Typography
+            className={classes.titleHeading}
+            variant="h1"
+            variantMapping={{ h1: 'span' }}
+          >
```

Comment:
Fixing Components to not render `<h1>` inside an `<h6>` element. Could be moved to seperate PR.